modules/themes/iFixedDiscord.js

```diff
@@ -0,0 +1,26 @@
+const version = "0";
+
+let style;
+
+let obj = {
+  onImport: async () => {
+    style = document.createElement("style");
+    document.head.appendChild(style);
+    style.appendChild(
+      document.createTextNode(
+        `@import url("https://raw.githack.com/j0lol/i-fixed-discord/main/css/theme.css")`
+      )
+    );
+  },
+
+  remove: async () => {
+    style.remove();
+  },
+
+  name: "i fixed discord",
+  description: "Use in light mode with dark sidebar enabled.",
```

Comment:
Maybe expand this description with what it is for / does?

Reply:
This is the title and description of the original theme, but yeah I should do that huh...